process/README.md

```diff
@@ -0,0 +1,30 @@
+
+# I. Overview
+
+This document describes the Open Source Security Foundation (OpenSSF) life cycle process for Technical Initiatives, both Working Groups, Projects, and Special Interest Groups (SIG).
+
+The authority that governs this process is as follows:
+
+The parent organizational structure grants governance to the downward in the organizational structure.  In turn the receiving Working Group, Project, or SIG reports health, participation, outcomes, statuses, etc up the chain.
```

Comment:
The intention and implications of this part are a bit unclear to me. Let's say a WG wants to create a new Project, do they have to submit a proposal to the TAC, or are they allowed to create the new Project on their own?

The `grants governance to the downward` statement seems to imply the WG can create new Projects, but the process below under project-lifecycle.md seems to imply that the TAC must review/approve all new Projects.

Reply:
I believe the intention is that if a WG has some small scripty source code they need to fulfill their charter, they should be able to put that source code under the OSSF, follow license and contributing guidelines, etc., and keep getting work done (line 20 in organizational_structure_overview). 

When it is something that is intended to amass users outside of the working group, needs foundation resources, or be promoted as an "OpenSSF Project," that's when it needs TAC approval.

Reply:
Perhaps a different way of phrasing it is: WG's may launch their own SIGs without TAC approval, and of course in their own repo might have miscellaneous code or other artifacts to assist with their chartered mission, but if they want to launch a new Project (capital-P) then they need TAC approval.

Reply:
> Perhaps a different way of phrasing it is: WG's may launch their own SIGs without TAC approval, and of course in their own repo might have miscellaneous code or other artifacts to assist with their chartered mission, but if they want to launch a new Project (capital-P) then they need TAC approval.

I think that's a reasonable middle ground but this calls for a clear definition of what is to be considered a Project.

What we don't want is to have a WG start developing some tool that ought to really be a Project and end up again with projects unaccounted for. This is not good for the project which needs to be known nor for the community which needs to know what's going on. 

And given that what a WG started as a small tool development may grow beyond the initial plan we ought to have directions on how to transition to a Project when appropriate

Reply:
I think this structure has a built-in control for these scenarios: a WG which decides to create a SIG should, at its next sync with the TAC, report up on that event, and make visible in its documentation that the SIG exists (including how to join, what its scope is, etc).

Reply:
> I think that's a reasonable middle ground but this calls for a clear definition of what is to be considered a Project.

If we're going to require TAC approval for Projects that live inside working groups, I think we need a clear definition of what constitutes a project vs. a prototype or small bit of code.

If we leave it to Working Groups, then I think we can avoid having to make that (probably blurry) distinction. 

I'm in favor of pushing as much of this down to the WGs as possible.

Reply:
> constitutes a project vs. a prototype or small bit of code

Perhaps: whether or not that code is in its own repo? "small scripty source code they need to fulfill their charter" should probably be in the WG's repo.